canonical-websites/www.ubuntu.com
# Add telco page

## Done

Add Telco page 

**Contact us link needed for 'hero' not to be merged until this is added**

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/telecommunications](http://0.0.0.0:8001/telecommunications)
- Make sure the page is as per [copy doc](https://docs.google.com/document/d/15FY_IaT39V81FQcoSoEJpmRRjBX80PJ7uxjgzHnDDws/edit)


## Issue / Card

Fixes #2390 
Doc: https://docs.google.com/document/d/15FY_IaT39V81FQcoSoEJpmRRjBX80PJ7uxjgzHnDDws/edit
